cosmpy/aerial/coins.py

```diff
@@ -427,17 +425,19 @@ def _math_operation(
         other: Union[
             str, "Coins", Iterable[Coin], Iterable[CoinProto], Coin, CoinProto
         ],
-        result_inout: "Coins",
+        result_input: "Coins",
     ):
-        res = result_inout
+        res = result_input
 
         if not isinstance(other, Coins):
             other = Coins(other)
 
         for c in other:
             left: Coin = res.get(c.denom, 0)
             yield left, c
-            res._merge_coin(left, on_collision=OnCollision.Override)
+            res._merge_coin(  # pylint: disable=protected-access
```

Comment:
The `# pylint: disable=protected-access` shall not be necessary, since the `Coins._math_operation(...)` method is defined in the scope of the `Coins` class where the `Coins._merge_coin(...)` methond in question is located as well.
I'm not getting any issue being reported by pylint here without the `# pylint: disable=protected-access`.

Reply:
This pylint disblement seems to be necessary to be present due to false positive - see [here](https://github.com/pylint-dev/pylint/issues/3607).